src/core/api.c

```diff
@@ -521,7 +521,7 @@ MsQuicConnectionSendResumptionTicket(
         goto Error;
     }
 
-    if (Flags > (QUIC_SEND_RESUMPTION_FLAG_FINAL | QUIC_SEND_RESUMPTION_FLAG_NONE)) {
+    if (Flags > (QUIC_SEND_RESUMPTION_FLAG_FINAL | QUIC_SEND_RESUMPTION_FLAG_NONE)) { // cppcheck-suppress badBitmaskCheck
```

Comment:
Really, the `| QUIC_SEND_RESUMPTION_FLAG_NONE` should just be removed here, right? Instead of ignoring?